### ISSUE TITLE
1.20.1 can not fly

### ISSUE THREAD

**Issue by hallool** [DenisMasterHerobrine/AngelRing#49]
hello i can not fly with any of the angel rings in survival im only getting the slow fall effect

**Comment by DenisMasterHerobrine** [comment]
I'll reopen this issue if I get more info and will be able to reproduce the issue.

**Comment by hallool** [comment]
yeah sry im using forge 47.1.3  bcs many mods only work on this version at this time     mod version   2.2.1  
solo world  custom pack     i found this but cant find the mod that couses slow faling instead of fyling..   
![grafik](https://github.com/DenisMasterHerobrine/AngelRing/assets/48066058/86414b45-03d4-4bdd-8d61-c6f54e812884)

**Comment by Riddler60** [comment]
So I had this same bug. I also found out it came down to 2 mods for me that caused this issue. [Portal's Superpowers](https://www.curseforge.com/minecraft/mc-mods/portals-superpowers) mod and [Create: Balanced Flight (Angel Ring)](https://github.com/anthxnymc/BalancedFlight) mod Create: Balanced Flight (Angel Ring) caused my slow falling effect and Portal's Superpowers wouldn't allow me to fly at all.

**Comment by kaschparl** [comment]
Hello,
encountered the same effect with my collection of mods. Didn't really made or used a modpack. Simply gathered some mods (285 or so) I liked to play on my own server in 1.20.1 forge version 47.4.6. 
Angel Ring version 2.3.1
Flight stopped everytime I changed the item in my hand or placed a block in the world.
Was a long run on testing mod by mod to find the one that's causing the problem. Found out that DeepMoblearning adds creative flight to the game, too and removed it. :(
